### ISSUE TITLE
Trying to revoke/interact with the last token in the list looks odd

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#18785]
1. You need to scroll to see all options (maybe the menu should float to the top at some point?)
2. The extended space is white instead of black on dark mode

![Bildschirmfoto von 2020-01-09 16-30-56](https://user-images.githubusercontent.com/213943/72080862-c61b1600-32fd-11ea-89d1-bce8b631a912.png)

cc @nextcloud/designers

**Comment by kesselb** [comment]
Here is another one: https://github.com/nextcloud/server/issues/18421 Good to have this fixed for 18.0.1.

**Comment by ma12-co** [comment]
I think that this nextcloud/nextcloud-vue#641
Would solve the problem

**Comment by jancborchardt** [comment]
This has been fixed in the component since the menus now open to the top when there is no space on the bottom:
![image](https://user-images.githubusercontent.com/925062/91090967-32147980-e656-11ea-8c79-f8d63e52c983.png)
